<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Option</class>
 <widget class="QWidget" name="Option">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1039</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1039</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Microsoft Sans Serif</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: rgb(230, 230, 230);
	background-color: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #07BEFF stop: 1 #2B74FF);
	alternate-background-color: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #07BEFF stop: 1 #2B74FF);
	border: 1px solid rgba(255, 255, 255, 0);
	border-radius: 16px;
}

QPushButton:hover {
    color: rgb(230, 230, 230);
	background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
	alternate-background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
	border: 1px solid rgba(255, 255, 255, 0);
	border-radius: 16px;
}

QPushButton:pressed {
	color: rgb(255, 255, 255);
    background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
  alternate-background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
  border-radius: 16px;
}</string>
   </property>
   <property name="text">
    <string>‚ù∞</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>23</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	alternate-background-color: rgba(0, 0, 0, 0);
	background-color: rgba(0, 0, 0, 0);
	color: rgb(227, 226, 228);
}</string>
   </property>
   <property name="text">
    <string>OPTION</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_box_realization">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>100</y>
     <width>150</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>190</width>
     <height>32</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	selection-color: rgb(255, 255, 255);
	background-color: rgba(0, 0, 0, 0.0);
	alternate-background-color: rgba(0, 0, 0, 0);
	color: rgba(255, 255, 255, 0.7);
	border: 1px solid rgba(255, 255, 255, 0);
	selection-background-color: rgba(0,0,0, 0);
}</string>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>Matrix</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Graph</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_open_config">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>280</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: rgb(230, 230, 230);
	background-color: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #73AF21 stop: 1 #598A1B);
	alternate-background-color: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #73AF21 stop: 1 #598A1B);
	border: 1px solid rgba(255, 255, 255, 0);
	border-radius: 14px;
}

QPushButton:hover {
    color: rgb(230, 230, 230);
	background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #8ED627 stop: 1 #6CA521);
	alternate-background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #8ED627 stop: 1 #6CA521);
	border-radius: 14px;
}

QPushButton:pressed {
	color: rgb(255, 255, 255);
    background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #8ED627 stop: 1 #6CA521);
  alternate-background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #8ED627 stop: 1 #6CA521);
  border-radius: 14px;
}

QPushButton:disabled {
	color: rgb(143,152,160);
	background-color: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #2C3C49 stop: 1 #53616F);
	alternate-background-color: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #2C3C49 stop: 1 #53616F);
	border: 1px solid rgba(0, 0, 0, 0);
	border-radius: 14px;
}</string>
   </property>
   <property name="text">
    <string>OPEN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_config_file_path">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>310</y>
     <width>430</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>430</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>430</width>
     <height>32</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	alternate-background-color: rgba(0, 0, 0, 0);
	background-color: rgba(0, 0, 0, 0);
	color: rgb(127, 126, 128);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_config_file">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>300</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>32</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>23</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	alternate-background-color: rgba(0, 0, 0, 0);
	background-color: rgba(0, 0, 0, 0);
	color: rgb(227, 226, 228);
}</string>
   </property>
   <property name="text">
    <string>Configuration file: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_realization">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>300</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>32</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>23</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	alternate-background-color: rgba(0, 0, 0, 0);
	background-color: rgba(0, 0, 0, 0);
	color: rgb(227, 226, 228);
}</string>
   </property>
   <property name="text">
    <string>Model implementation: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_os">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>430</y>
     <width>300</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>32</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	alternate-background-color: rgba(0, 0, 0, 0);
	background-color: rgba(0, 0, 0, 0);
	color: rgb(227, 226, 228);
}</string>
   </property>
   <property name="text">
    <string>Operating system: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_kernel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>460</y>
     <width>300</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>32</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	alternate-background-color: rgba(0, 0, 0, 0);
	background-color: rgba(0, 0, 0, 0);
	color: rgb(227, 226, 228);
}</string>
   </property>
   <property name="text">
    <string>Kernel type: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_cpu">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>490</y>
     <width>300</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>32</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	alternate-background-color: rgba(0, 0, 0, 0);
	background-color: rgba(0, 0, 0, 0);
	color: rgb(227, 226, 228);
}</string>
   </property>
   <property name="text">
    <string>CPU: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_host">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>300</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>32</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	alternate-background-color: rgba(0, 0, 0, 0);
	background-color: rgba(0, 0, 0, 0);
	color: rgb(227, 226, 228);
}</string>
   </property>
   <property name="text">
    <string>Host name: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_cpu_value">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>490</y>
     <width>510</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>510</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>510</width>
     <height>32</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	alternate-background-color: rgba(0, 0, 0, 0);
	background-color: rgba(0, 0, 0, 0);
	color: rgb(227, 226, 228);
}</string>
   </property>
   <property name="text">
    <string>proc</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_kernel_value">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>460</y>
     <width>510</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>510</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>510</width>
     <height>32</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	alternate-background-color: rgba(0, 0, 0, 0);
	background-color: rgba(0, 0, 0, 0);
	color: rgb(227, 226, 228);
}</string>
   </property>
   <property name="text">
    <string>kernel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_os_value">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>430</y>
     <width>510</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>510</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>510</width>
     <height>32</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	alternate-background-color: rgba(0, 0, 0, 0);
	background-color: rgba(0, 0, 0, 0);
	color: rgb(227, 226, 228);
}</string>
   </property>
   <property name="text">
    <string>macos</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_host_value">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>400</y>
     <width>510</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>510</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>510</width>
     <height>32</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	alternate-background-color: rgba(0, 0, 0, 0);
	background-color: rgba(0, 0, 0, 0);
	color: rgb(227, 226, 228);
}</string>
   </property>
   <property name="text">
    <string>host</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_layers">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>300</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>32</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>23</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	alternate-background-color: rgba(0, 0, 0, 0);
	background-color: rgba(0, 0, 0, 0);
	color: rgb(227, 226, 228);
}</string>
   </property>
   <property name="text">
    <string>Perceptron layers: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_box_layers">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>160</y>
     <width>150</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>190</width>
     <height>32</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	selection-color: rgb(255, 255, 255);
	background-color: rgba(0, 0, 0, 0.0);
	alternate-background-color: rgba(0, 0, 0, 0);
	color: rgba(255, 255, 255, 0.7);
	border: 1px solid rgba(255, 255, 255, 0);
	selection-background-color: rgba(0,0,0, 0);
}</string>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>2 layers</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3 layers</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4 layers</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5 layers</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="combo_box_weights_type">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>220</y>
     <width>150</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>190</width>
     <height>32</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	selection-color: rgb(255, 255, 255);
	background-color: rgba(0, 0, 0, 0.0);
	alternate-background-color: rgba(0, 0, 0, 0);
	color: rgba(255, 255, 255, 0.7);
	border: 1px solid rgba(255, 255, 255, 0);
	selection-background-color: rgba(0,0,0, 0);
}</string>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>Random</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Load file</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_realization_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>300</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>32</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>23</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	alternate-background-color: rgba(0, 0, 0, 0);
	background-color: rgba(0, 0, 0, 0);
	color: rgb(227, 226, 228);
}</string>
   </property>
   <property name="text">
    <string>Perceptron weights mode: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
